taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -32,9 +32,11 @@ export const StartProject = () => {
 
   const findDefaultBranchName = useCallback(() => {
     if (taxonomyName === "" || ownerName === "") return "";
-    return `${toSnakeCase(taxonomyName.toLowerCase())}_${ownerName
-      .replace(" ", "")
-      .toLowerCase()}_${Math.floor(Date.now() / 1000)}`;
+    return `${toSnakeCase(
+      taxonomyName.toLowerCase()
+    )}_${ownerName}_${Math.floor(Date.now() / 1000)}`
```

Comment:
It would be nice to make the date more readable for a human (maybe something like MMDDYY_HH_MinMin? for example for today: 032724_12_13)